padl/transforms.py

```diff
@@ -749,6 +776,9 @@ def _gen():
 
         return _GeneratorWithLength(_gen, length)
 
+    def pd_set_device(self, device: str):
```

Comment:
@sjrl I'd suggest making this a setter for the `pd_device` property (`@pd_device.setter`)